### ISSUE TITLE
ember-fetch@^6.0.0

### ISSUE THREAD

**Issue by theseyi** [simplabs/ember-simple-auth#1705]
`ember-fetch@^6.0.0` has been released, `ember-simple-auth` should upgrade to and/or support this version:

```json
"dependencies": {
  "ember-fetch": "^2.1.0 || ^3.0.0 || ^4.0.0 || ^5.0.0",
}
```
should be:
```json
"dependencies": {
  "ember-fetch": "^2.1.0 || ^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0",
}
```

**Comment by andreyfel** [comment]
This blocks upgrading projects with mono-repo to ember-cli 3.5 because of the bug fixed in ember-fetch 6.1.1.

**Comment by andreyfel** [comment]
Created pr #1713 to address that issue.
